packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -245,4 +257,54 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
       allowMultiLogin: this.$config?.allowMultiLogin ?? true,
     };
   }
+
+  /**
+   * Called when we reconnection has been completed
+   */
+  private async handleConnectionLost(msg: ConnectionLostDetails): Promise<void> {
```

Comment:
This is an event handler when we lose connection in the web socket. This gets called and it internally will handle things such as overall event emission to the user that the socket has been disconnected or has been reconnected.
Here also we will call `silentRelogin` if the config allows for it.